From 2ecdf74b9caefd742216eea74daa24b2616a2ca4 Mon Sep 17 00:00:00 2001
From: Frodo Lai <frodo_lai@bcmcom.com>
Date: Thu, 18 May 2017 11:38:16 -0700
Subject: [PATCH 12/12] Remove unused imx6q-da61mx_lvds13.dts file

- Consolidate to one dts file after implementing LVDS auto
  selection.

Signed-off-by: Frodo Lai <frodo_lai@bcmcom.com>
---
 arch/arm/boot/dts/Makefile                |   1 -
 arch/arm/boot/dts/imx6q-da61mx_lvds13.dts | 623 ------------------------------
 2 files changed, 624 deletions(-)
 delete mode 100755 arch/arm/boot/dts/imx6q-da61mx_lvds13.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index cd29e09..455888c 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -173,7 +173,6 @@ dtb-$(CONFIG_ARCH_MXC) += \
 	imx6q-gr60mxd.dtb \
 	imx6dl-da61mx.dtb \
 	imx6q-da61mx.dtb \
-	imx6q-da61mx_lvds13.dtb \
 	imx6q-sabreauto.dtb \
 	imx6q-sabreauto-flexcan1.dtb \
 	imx6q-sabreauto-ecspi.dtb \
diff --git a/arch/arm/boot/dts/imx6q-da61mx_lvds13.dts b/arch/arm/boot/dts/imx6q-da61mx_lvds13.dts
deleted file mode 100755
index 4e9f527..0000000
--- a/arch/arm/boot/dts/imx6q-da61mx_lvds13.dts
+++ /dev/null
@@ -1,623 +0,0 @@
-/* For BCMTWN DA61MX416Q.
- */
-
-/dts-v1/;
-
-#include <dt-bindings/gpio/gpio.h>
-#include <dt-bindings/input/input.h>
-#include "imx6q.dtsi"
-
-/ {
-	model = "BCM DA61MX416Q Board";
-	compatible = "bcmtwn,imx6q-da61mx", "fsl,imx6q";
-
-	aliases {
-		mxcfb0 = &mxcfb1;
-		/*
-		mxcfb1 = &mxcfb2;
-		mxcfb2 = &mxcfb3;
-		mxcfb3 = &mxcfb4;
-		*/
-	};
-
-	memory:	memory {
-		reg = <0x10000000 0xF0000000>;
-	};
-
-	regulators {
-		compatible = "simple-bus";
-
-		reg_usb_otg_vbus: usb_otg_vbus {
-			compatible = "regulator-fixed";
-			regulator-name = "usb_otg_vbus";
-			regulator-min-microvolt = <5000000>;
-			regulator-max-microvolt = <5000000>;
-			/*
-			regulator-always-on;
-            */
-			gpio = <&gpio5 14 GPIO_ACTIVE_HIGH>;
-            enable-active-high;
-		};
-
-		reg_usb_h1_vbus: usb_h1_vbus {
-			compatible = "regulator-fixed";
-			regulator-name = "usb_h1_vbus";
-			regulator-min-microvolt = <5000000>;
-			regulator-max-microvolt = <5000000>;
-			regulator-always-on;
-			/*
-			gpio = <&gpio5 14 GPIO_ACTIVE_HIGH>;
-            enable-active-high;
-			*/
-		};
-
-	    bt_rfkill {
-            compatible = "fsl,mxc_bt_rfkill";
-            status ="okay";
-	    };
-	};
-
-	sound {
-		compatible = "fsl,bcmtwn-imx-audio-wm8960";
-		model = "wm8960-audio";
-		cpu-dai = <&ssi2>;
-		audio-codec = <&codec>;
-		codec-master;
-		mux-int-port = <2>;
-		mux-ext-port = <3>;
-		/*
-    	asrc-controller = <&asrc>;
-		*/
-
-		audio-routing =
-			"Headphone Jack", "HP_L",
-			"Headphone Jack", "HP_R";
-	};
-
-	leds {
-		compatible = "gpio-leds";
-
-		status-led {
-			label = "PowerOn";
-			gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "default-on";
-			retain-state-suspended;
-			default-state = "on";
-		};
-
-		sata-led {
-			label = "SATA-Activity";
-			gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
-			linux,default-trigger = "ide-disk";
-			default-state = "off";
-		};
-
-		usb-charger-led {
-			label = "USB-Charger-LED";
-			gpios = <&gpio4 11 GPIO_ACTIVE_LOW>;
-			linux,default-trigger = "charger";
-			retain-state-suspended;
-			default-state = "off";
-		};
-	};
-
-	gpio-keys {
-		compatible = "gpio-keys";
-		pinctrl-names = "default";
-		pinctrl-0 = <&pinctrl_gpio_keys>;
-
-		power {
-			label = "Power Button";
-			gpios = <&gpio1 2 GPIO_ACTIVE_LOW>;
-			gpio-key,wakeup;
-			linux,code = <KEY_POWER>;
-		};
-	};
-
-	/* Setting for 10" LVDS -START */
-	/*
-	mxcfb1: fb@0 {
-		compatible = "fsl,mxc_sdc_fb";
-		disp_dev = "ldb";
-		interface_pix_fmt = "RGB24";
-		default_bpp = <32>;
-		int_clk = <0>;
-		late_init = <0>;
-		status = "okay";
-	};
-
-	pwm-backlight {
-		compatible = "pwm-backlight";
-		pwms = <&pwm1 0 20000>;
-		brightness-levels = <0 10 20 30 40 50 60 70 80 90 100>;
-		default-brightness-level = <8>;
-		enable-gpios = <&gpio1 13 GPIO_ACTIVE_HIGH>;
-	};
-	*/
-	/* Setting for 10" LVDS -END */
-
-	/* Setting for 13" LVDS -START */
-	mxcfb1: fb@0 {
-		compatible = "fsl,mxc_sdc_fb";
-		disp_dev = "ldb";
-		interface_pix_fmt = "RGB24";
-		default_bpp = <32>;
-		int_clk = <0>;
-		late_init = <0>;
-		status = "okay";
-	};
-
-	pwm-backlight {
-		compatible = "pwm-backlight";
-		pwms = <&pwm2 0 5000>;
-		brightness-levels = <0 10 20 30 40 50 60 70 80 90 100>;
-		default-brightness-level = <8>;
-		enable-gpios = <&gpio1 12 GPIO_ACTIVE_HIGH>;
-	};
-	/* Setting for 13" LVDS -END */
-
-	v4l2_out {
-		compatible = "fsl,mxc_v4l2_output";
-		status = "okay";
-	};
-
-	ramoops_device {
-		compatible = "fsl,mxc_ramoops";
-		record_size = <524288>; /*512K*/
-		console_size= <262144>; /*256K*/
-		ftrace_size= <262144>;  /*256K*/
-		dump_oops = <1>;
-		status = "okay";
-	};
-};
-
-&clks {
-	fsl,ldb-di0-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
-	fsl,ldb-di1-parent = <&clks IMX6QDL_CLK_PLL2_PFD0_352M>;
-};
-
-&iomuxc {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_da61mx416q>;
-
-	da61mx416q {
-		pinctrl_da61mx416q: da61mx416qgrp {
-			fsl,pins = <
-				MX6QDL_PAD_SD2_CMD__GPIO1_IO11		0x80000000	/* L0_PWREN */
-				MX6QDL_PAD_SD2_CLK__GPIO1_IO10		0x80000000	/* L1_PWREN */
-				MX6QDL_PAD_SD2_DAT0__GPIO1_IO15		0x80000000	/* BL0_PWREN */
-				MX6QDL_PAD_SD2_DAT1__GPIO1_IO14		0x80000000	/* BL1_PWREN */
-				MX6QDL_PAD_SD2_DAT2__GPIO1_IO13		0x80000000	/* BL0EN */
-				MX6QDL_PAD_SD2_DAT3__GPIO1_IO12		0x80000000	/* BL1EN */
-				MX6QDL_PAD_KEY_COL4__GPIO4_IO14		0x80000000	/* UD_24V_OUT */
-				MX6QDL_PAD_KEY_ROW4__GPIO4_IO15		0x80000000	/* REC_MOD_SEL */
-				MX6QDL_PAD_GPIO_4__GPIO1_IO04		0x80000000	/* STATUS_LED */
-				MX6QDL_PAD_GPIO_5__GPIO1_IO05		0x80000000	/* SATA_LED?? */
-				MX6QDL_PAD_GPIO_7__GPIO1_IO07		0x80000000	/* PANEL_SEL */
-				MX6QDL_PAD_GPIO_8__GPIO1_IO08		0x80000000	/* NFC_ENABLE */
-				MX6QDL_PAD_GPIO_9__GPIO1_IO09		0x80000000	/* NFC_IRQ */
-				MX6QDL_PAD_GPIO_17__GPIO7_IO12		0x80000000	/* PCIE_RST_B */
-				MX6QDL_PAD_GPIO_18__GPIO7_IO13		0x80000000	/* PCIE_DIS_B */
-				MX6QDL_PAD_GPIO_19__GPIO4_IO05		0x80000000	/* WIFI_PD# */
-				MX6QDL_PAD_NANDF_CS0__GPIO6_IO11	0x80000000	/* TUH_RST_B */
-				MX6QDL_PAD_NANDF_CS1__GPIO6_IO14	0x80000000	/* NFC_MOD_FD */
-				MX6QDL_PAD_NANDF_ALE__GPIO6_IO08	0x80000000	/* CAP_TCH_INT0 */
-				MX6QDL_PAD_NANDF_D0__GPIO2_IO00		0x80000000	/* SW_RST_IN# */
-				MX6QDL_PAD_NANDF_D1__GPIO2_IO01		0x80000000	/* SENSOR_INT */
-				MX6QDL_PAD_NANDF_D2__GPIO2_IO02		0x80000000	/* FRAM_WP */
-				MX6QDL_PAD_DISP0_DAT4__GPIO4_IO25	0x80000000	/* VER_B0 */
-				MX6QDL_PAD_DISP0_DAT5__GPIO4_IO26	0x80000000	/* VER_B1 */
-				MX6QDL_PAD_DISP0_DAT6__GPIO4_IO27	0x80000000	/* VER_B2 */
-				MX6QDL_PAD_DISP0_DAT13__GPIO5_IO07	0x80000000	/* CHG_CON_5VOC */
-				MX6QDL_PAD_DISP0_DAT19__GPIO5_IO13	0x80000000	/* CHG_CON_5VEN */
-				MX6QDL_PAD_DISP0_DAT20__GPIO5_IO14	0x80000000	/* USBH_PW_EN */
-				MX6QDL_PAD_CSI0_DAT11__GPIO5_IO29	0x80000000	/* PHY_RST */
-				MX6QDL_PAD_CSI0_MCLK__CCM_CLKO1    	0x130b0		/* AUD_MCLK */
-			>;
-		};
-
-		pinctrl_uart4: uart4grp {
-			fsl,pins = <
-				MX6QDL_PAD_KEY_COL0__UART4_TX_DATA	0x1b0b1
-				MX6QDL_PAD_KEY_ROW0__UART4_RX_DATA	0x1b0b1
-			>;
-		};
-
-		pinctrl_pwm1: pwm1grp-1 {
-			fsl,pins = <
-				MX6QDL_PAD_DISP0_DAT8__PWM1_OUT 0x1b0b1
-			>;
-		};
-
-		pinctrl_pwm2: pwm2grp-1 {
-			fsl,pins = <
-				MX6QDL_PAD_DISP0_DAT9__PWM2_OUT 0x1b0b1
-			>;
-		};
-
-		pinctrl_enet: enetgrp {
-			fsl,pins = <
-				MX6QDL_PAD_ENET_MDIO__ENET_MDIO	        0x1b0b0
-				MX6QDL_PAD_ENET_MDC__ENET_MDC	        0x1b0b0
-				MX6QDL_PAD_GPIO_16__ENET_REF_CLK		0x1b0b0
-				MX6QDL_PAD_ENET_RXD0__ENET_RX_DATA0		0x1b0b0
-				MX6QDL_PAD_ENET_RXD1__ENET_RX_DATA1		0x1b0b0
-				MX6QDL_PAD_ENET_CRS_DV__ENET_RX_EN		0x1b0b0
-				MX6QDL_PAD_ENET_TXD0__ENET_TX_DATA0		0x1b0b0
-				MX6QDL_PAD_ENET_TXD1__ENET_TX_DATA1		0x1b0b0
-				MX6QDL_PAD_ENET_TX_EN__ENET_TX_EN		0x1b0b0
-				MX6QDL_PAD_ENET_RX_ER__ENET_RX_ER		0x1b0b0
-			>;
-		};
-
-		pinctrl_i2c1: i2c1grp {
-			fsl,pins = <
-				MX6QDL_PAD_CSI0_DAT8__I2C1_SDA		0x4001b8b1
-				MX6QDL_PAD_CSI0_DAT9__I2C1_SCL		0x4001b8b1
-			>;
-		};
-
-		pinctrl_i2c2: i2c2grp {
-			fsl,pins = <
-				MX6QDL_PAD_KEY_COL3__I2C2_SCL		0x4001b8b1
-				MX6QDL_PAD_KEY_ROW3__I2C2_SDA		0x4001b8b1
-			>;
-		};
-
-		pinctrl_i2c3: i2c3grp {
-			fsl,pins = <
-				MX6QDL_PAD_GPIO_3__I2C3_SCL		0x4001b8b1
-				MX6QDL_PAD_GPIO_6__I2C3_SDA		0x4001b8b1
-			>;
-		};
-
-		pinctrl_usbotg: usbotggrp {
-			fsl,pins = <
-				MX6QDL_PAD_GPIO_1__USB_OTG_ID	0x17059
-			>;
-		};
-
-		pinctrl_wifi: wifigrp {
-			fsl,pins = <
-				MX6QDL_PAD_SD1_CMD__SD1_CMD			0x17059
-				MX6QDL_PAD_SD1_CLK__SD1_CLK			0x10059
-				MX6QDL_PAD_SD1_DAT0__SD1_DATA0		0x17059
-				MX6QDL_PAD_SD1_DAT1__SD1_DATA1		0x17059
-				MX6QDL_PAD_SD1_DAT2__SD1_DATA2		0x17059
-				MX6QDL_PAD_SD1_DAT3__SD1_DATA3		0x17059
-			>;
-		};
-
-		pinctrl_wifi_100mhz: wifigrp100mhz {
-			fsl,pins = <
-				MX6QDL_PAD_SD1_CMD__SD1_CMD			0x170b9
-				MX6QDL_PAD_SD1_CLK__SD1_CLK			0x100b9
-				MX6QDL_PAD_SD1_DAT0__SD1_DATA0		0x170b9
-				MX6QDL_PAD_SD1_DAT1__SD1_DATA1		0x170b9
-				MX6QDL_PAD_SD1_DAT2__SD1_DATA2		0x170b9
-				MX6QDL_PAD_SD1_DAT3__SD1_DATA3		0x170b9
-			>;
-		};
-
-		pinctrl_wifi_200mhz: wifigrp200mhz {
-			fsl,pins = <
-				MX6QDL_PAD_SD1_CMD__SD1_CMD			0x170f9
-				MX6QDL_PAD_SD1_CLK__SD1_CLK			0x100f9
-				MX6QDL_PAD_SD1_DAT0__SD1_DATA0		0x170f9
-				MX6QDL_PAD_SD1_DAT1__SD1_DATA1		0x170f9
-				MX6QDL_PAD_SD1_DAT2__SD1_DATA2		0x170f9
-				MX6QDL_PAD_SD1_DAT3__SD1_DATA3		0x170f9
-			>;
-		};
-
-		/*
-		pinctrl_usdhc3: usdhc3grp {
-			fsl,pins = <
-				MX6QDL_PAD_SD3_CMD__SD3_CMD			0x17059
-				MX6QDL_PAD_SD3_CLK__SD3_CLK			0x10059
-				MX6QDL_PAD_SD3_DAT0__SD3_DATA0		0x17059
-				MX6QDL_PAD_SD3_DAT1__SD3_DATA1		0x17059
-				MX6QDL_PAD_SD3_DAT2__SD3_DATA2		0x17059
-				MX6QDL_PAD_SD3_DAT3__SD3_DATA3		0x17059
-			>;
-		};
-		*/
-
-		pinctrl_usdhc4: usdhc4grp {
-			fsl,pins = <
-				MX6QDL_PAD_SD4_CMD__SD4_CMD			0x17059
-				MX6QDL_PAD_SD4_CLK__SD4_CLK			0x10059
-				MX6QDL_PAD_SD4_DAT0__SD4_DATA0		0x17059
-				MX6QDL_PAD_SD4_DAT1__SD4_DATA1		0x17059
-				MX6QDL_PAD_SD4_DAT2__SD4_DATA2		0x17059
-				MX6QDL_PAD_SD4_DAT3__SD4_DATA3		0x17059
-				MX6QDL_PAD_SD4_DAT4__SD4_DATA4		0x17059
-				MX6QDL_PAD_SD4_DAT5__SD4_DATA5		0x17059
-				MX6QDL_PAD_SD4_DAT6__SD4_DATA6		0x17059
-				MX6QDL_PAD_SD4_DAT7__SD4_DATA7		0x17059
-			>;
-		};
-
-
-		pinctrl_audmux: audmuxgrp {
-			fsl,pins = <
-				MX6QDL_PAD_CSI0_DAT7__AUD3_RXD		0x130b0
-				MX6QDL_PAD_CSI0_DAT4__AUD3_TXC		0x130b0
-				MX6QDL_PAD_CSI0_DAT5__AUD3_TXD		0x110b0
-				MX6QDL_PAD_CSI0_DAT6__AUD3_TXFS		0x130b0
-			>;
-		};
-
-		pinctrl_gpio_keys: gpio_keysgrp {
-			fsl,pins = <
-				MX6QDL_PAD_GPIO_2__GPIO1_IO02		0x1b0b0	/* SW_MR_RST */
-			>;
-		};
-	};
-
-	gpio_leds {
-		pinctrl_gpio_leds: gpioledsgrp {
-			fsl,pins = <
-				MX6QDL_PAD_GPIO_4__GPIO1_IO04 		0x1b0b0		/* STATUS_LED */
-				MX6QDL_PAD_GPIO_5__GPIO1_IO05 		0x1b0b0		/* SATA_LED */
-				MX6QDL_PAD_KEY_ROW2__GPIO4_IO11		0x1b0b0		/* CHG_CON_BLPW_EN */
-			>;
-		};
-	};
-};
-
-&i2c1 {
-	clock-frequency = <400000>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c1>;
-	status = "okay";
-
-	codec: wm8960@1a {
-		compatible = "wlf,wm8960";
-		reg = <0x1a>;
-		wlf,shared-lrclk;
-		clocks = <&clks IMX6QDL_CLK_CKO>;
-		clock-names = "mclk";
-		status = "okay";
-	};
-};
-
-&i2c2 {
-	clock-frequency = <400000>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c2>;
-	status = "okay";
-
-	/*
-	egalax_ts@2d {
-		compatible = "eeti,egalax_ts";
-		reg = <0x2d>;
-		interrupt-parent = <&gpio6>;
-		interrupts = <8 IRQ_TYPE_EDGE_FALLING>;
-		wakeup-gpios = <&gpio6 8 GPIO_ACTIVE_HIGH>;
-	};
-	*/
-
-	nt3h1x01: nt3h1x01@55 {
-		compatible = "nxp,nt3h1x01";
-		reg = <0x55>;
-		interrupt-gpios = <&gpio6 14 IRQ_TYPE_EDGE_RISING>;
-	};
-};
-
-&i2c3 {
-	clock-frequency = <400000>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_i2c3>;
-	status = "okay";
-
-	fm24w256: eeprom@56 {
-		compatible = "atmel,24c256", "cypress,fm24w256";
-		reg = <0x56>;
-		pagesize = <8>;
-	};
-
-	lsm6ds3@6b {
-		compatible = "st,lsm6ds3";
-		reg = <0x6b>;
-		interrupt-parent = <&gpio2>;
-		interrupts = <1 IRQ_TYPE_EDGE_RISING>;
-	};
-};
-
-&fec {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_enet>;
-	phy-mode = "rmii";
-	phy-reset-gpios = <&gpio5 29 GPIO_ACTIVE_LOW>;
-	status = "okay";
-};
-
-/* Setting for 10" LVDS - START */
-/*
-&ldb {
-	status = "okay";
-
-	lvds-channel@0 {
-		crtc = "ipu2-di0";
-		fsl,data-mapping = "spwg";
-		fsl,data-width = <24>;
-		primary;
-		status = "okay";
-
-		display-timings {
-			native-mode = <&timing0>;
-			timing0: G101EVN01 {
-				clock-frequency = <68930000>;
-				hactive = <1280>;
-				vactive = <800>;
-				hback-porch = <68>;
-				hfront-porch = <0>;
-				vback-porch = <6>;
-				vfront-porch = <0>;
-				hsync-len = <60>;
-				vsync-len = <10>;
-			};
-		};
-	};
-};
-
-&dcic1 {
-	dcic_id = <0>;
-	dcic_mux = "dcic-lvds0";
-	status = "okay";
-};
-
-&pwm1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_pwm1>;
-	status = "okay";
-};
-*/
-/* Setting for 10" LVDS - END */
-
-/* Setting for 13" LVDS - START */
-&ldb {
-	status = "okay";
-	split-mode;
-
-	lvds-channel@0 {
-		crtc = "ipu2-di0";
-		fsl,data-mapping = "jeida";
-		fsl,data-width = <24>;
-		primary;
-		status = "okay";
-
-		display-timings {
-			native-mode = <&timing0>;
-			timing0: G133HAN01 {
-				clock-frequency = <141200000>;
-				hactive = <1920>;
-				vactive = <1080>;
-				hback-porch = <146>;
-				hfront-porch = <0>;
-				vback-porch = <22>;
-				vfront-porch = <0>;
-				hsync-len = <42>;
-				vsync-len = <14>;
-			};
-		};
-	};
-};
-
-&dcic1 {
-	dcic_id = <0>;
-	dcic_mux = "dcic-lvds0";
-	status = "okay";
-};
-
-&dcic2 {
-	dcic_id = <1>;
-	dcic_mux = "dcic-lvds1";
-	status = "okay";
-};
-
-&pwm2 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_pwm2>;
-	status = "okay";
-};
-/* Setting for 13" LVDS - END */
-
-&pcie {
-	reset-gpio = <&gpio7 12 GPIO_ACTIVE_LOW>;
-	disable-gpio = <&gpio7 13 GPIO_ACTIVE_HIGH>;
-	status = "okay";
-};
-
-&uart4 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_uart4>;
-	status = "okay";
-};
-
-&usbh1 {
-	vbus-supply = <&reg_usb_h1_vbus>;
-	status = "okay";
-};
-
-&usbotg {
-	vbus-supply = <&reg_usb_otg_vbus>;
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_usbotg>;
-	disable-over-current;
-	srp-disable;
-	hnp-disable;
-	adp-disable;
-	status = "okay";
-};
-
-&usdhc1 {
-	pinctrl-names = "default", "state_100mhz", "state_200mhz";
-	pinctrl-0 = <&pinctrl_wifi>;
-	pinctrl-1 = <&pinctrl_wifi_100mhz>;
-	pinctrl-2 = <&pinctrl_wifi_200mhz>;
-	bus-width = <4>;
-	cap-sdio-irq;
-	/* cap-power-off-card; */
-	/* sd-uhs-sdr104; */
-	/* sd-uhs-sdr50; */
-	/* wifi-host; */
-	non-removable;
-	keep-power-in-suspend;
-	enable-sdio-wakeup;
-	status = "okay";
-};
-
-/*
-&usdhc3 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_usdhc3>;
-	cd-gpios = <&gpio7 0 GPIO_ACTIVE_LOW>;
-	no-1-8-v;
-	keep-power-in-suspend;
-	enable-sdio-wakeup;
-	status = "okay";
-};
-*/
-
-&usdhc4 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_usdhc4>;
-	bus-width = <8>;
-	non-removable;
-	no-1-8-v;
-	keep-power-in-suspend;
-	status = "okay";
-};
-
-&ssi2 {
-	status = "okay";
-};
-
-&ssi3 {
-	status = "okay";
-};
-
-&audmux {
-	pinctrl-names = "default";
-	pinctrl-0 = <&pinctrl_audmux>;
-	status = "okay";
-};
-
-&sata {
-	status = "okay";
-};
-
-&caam_sm  {
-	status = "okay";
-};
-
-&caam_snvs  {
-	status = "okay";
-};
-
-&crypto {
-	status = "okay";
-};
-
-- 
2.7.4

